frontend/src/components/payments/ViewPayments.vue

```diff
@@ -0,0 +1,156 @@
+
+    <v-skeleton-loader :loading="isLoading" type="table-tbody">
+      <v-data-table
+        :headers="paymentTableHeaders"
+        :items="displayedPayments"
+        :items-per-page="20"
+        :items-per-page-options="[10, 20, 50]"
+        :mobile="null"
+        mobile-breakpoint="md"
+        class="elevation-2"
+      >
+        <template #item.paymentPeriod="{ item }">
+          {{ formatMonthYearToString(item.paymentMonth, item.paymentYear) }}
+        </template>
+
+        <template #item.paymentAmount="{ item }"> {{ formatCurrency(item.paymentAmount) }} </template>
+
+        <template #[`item.approvedDate`]="{ item }">
+          {{ formatUTCDate(item.approvedDate) }}
+        </template>
+
+        <template #[`item.paidDate`]="{ item }">
+          {{ formatUTCDate(item.paidDate) }}
+        </template>
+
+        <template #[`item.paymentStatusCode`]="{ item }">
+          <span :class="getStatusClass(item.paymentStatusCode)">
+            {{ getDisplayStatus(item.paymentStatusCode) }}
+          </span>
+        </template>
+      </v-data-table>
+    </v-skeleton-loader>
+  </v-container>
+</template>
+
+<script>
+import { mapState } from 'pinia';
+
+import alertMixin from '@/mixins/alertMixin.js';
+
+import PaymentService from '@/services/paymentService.js';
+
+import { useApplicationStore } from '@/store/application.js';
+import { useOrganizationStore } from '@/store/ccof/organization.js';
+
+import { PAYMENT_STATUSES, PAYMENT_STATUS_TEXTS } from '@/utils/constants.js';
+import { formatCurrency, formatMonthYearToString, formatUTCDate } from '@/utils/format';
+
+export default {
+  name: 'ViewPayments',
+  mixins: [alertMixin],
+  data() {
+    return {
+      isLoading: false,
+      payments: [],
+      paymentTableHeaders: [
+        { title: 'Facility Name', sortable: true, value: 'facilityName' },
+        { title: 'Facility ID', sortable: true, value: 'facilityId' },
+        { title: 'Licence Number', sortable: true, value: 'licenceNumber' },
+        { title: 'Month of Service', sortable: true, value: 'paymentPeriod', width: '200px' },
+        { title: 'Funding Type', sortable: true, value: 'fundingTypeText' },
+        { title: 'Base/Adjustment Report', sortable: true, value: 'reportTypeText' },
+        { title: 'Amount($)', sortable: true, value: 'paymentAmount' },
+        { title: 'Approved Date', sortable: true, value: 'approvedDate' },
+        { title: 'Paid Date', sortable: true, value: 'paidDate', width: '150px' },
+        { title: 'Invoice Number', sortable: true, value: 'invoiceNumber', width: '150px' },
+        { title: 'Payment Line Status', sortable: true, value: 'paymentStatusCode' },
+      ],
+    };
+  },
+  computed: {
+    ...mapState(useOrganizationStore, ['organizationId']),
+    ...mapState(useApplicationStore, ['programYearId']),
+
+    displayedPayments() {
+      return this.payments.filter((payment) => payment.paymentAmount !== 0);
+    },
```

Comment:
Since the payments don't change after being loaded and we don't need the zero dollar payments I would just filter this.payments itself in loadPayments().